src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -160,20 +172,20 @@ public override bool Execute()
                     {
                         if (response.IsSuccessStatusCode)
                         {
-                            // Blobs can be files but have the name of a directory.  We'll skip those and log something weird happened.
-                            if (!string.IsNullOrEmpty(Path.GetFileName(filename)))
-                            {
-                                Stream responseStream = await response.Content.ReadAsStreamAsync();
+                                // Blobs can be files but have the name of a directory.  We'll skip those and log something weird happened.
+                                if (!string.IsNullOrEmpty(Path.GetFileName(filename)))
+                                {
+                                    Stream responseStream = await response.Content.ReadAsStreamAsync();
 
-                                using (FileStream sourceStream = File.Open(filename, FileMode.Create))
+                                    using (FileStream sourceStream = File.Open(filename, FileMode.Create))
+                                    {
+                                        responseStream.CopyTo(sourceStream);
+                                    }
+                                }
+                                else
                                 {
-                                    responseStream.CopyTo(sourceStream);
+                                    Log.LogWarning($"Unable to download blob '{blob}' as it has a directory-like name. This may cause problems if it was needed.");
                                 }
-                            }
-                            else
-                            {
-                                Log.LogWarning($"Unable to download blob '{blob}' as it has a directory-like name.  This may cause problems if it was needed.");
-                            }
```

Comment:
Has anything changed in these lines except the extra tab (which should be removed)?

Reply:
Ugh, was dealing with a new install of visual studio. seems to have messed with the formatting.